src/lv_conf_kconfig.h

```diff
@@ -18,6 +18,15 @@ extern "C" {
 
 #  ifdef __NuttX__
 #    include <nuttx/config.h>
+/*
+ * Make sure version number in Kconfig file is correctly set.
+ * Mismatch can happen when user manually copy lvgl/Kconfig file to their project, like what NuttX does.
+ */
+#    include "../lvgl/lv_version.h"
+
+#    if CONFIG_LVGL_VERSION_MAJOR != LVGL_VERSION_MAJOR || CONFIG_LVGL_VERSION_MINOR != LVGL_VERSION_MINOR
```

Comment:
I did not compare PATCH version. Please suggest if is needed.

Reply:
I think matching major and minor should solve most of the issues. Theoretically, it can happen that we fix a Kconfig entry in a patch release but it's very rare. It would happen more often that people need to update Kconfig for nothing.

Reply:
Added PATCH version check to be sure.